tests/release/pipelines/rh_push_to_external_registry.go

```diff
@@ -51,13 +55,25 @@ var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-pu
 	var snapshotPush *appservice.Snapshot
 	var releaseCR *releaseApi.Release
 
+	var testEnvironment = utils.GetEnv("TEST_ENVIRONMENT", releasecommon.UpstreamTestEnvironment)
+
 	BeforeAll(func() {
-		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("push-pyxis"))
-		Expect(err).NotTo(HaveOccurred())
-		devNamespace = fw.UserNamespace
-		managedNamespace = utils.GetGeneratedNamespace("push-pyxis-managed")
+		if testEnvironment == releasecommon.DownstreamTestEnvironment {
+			fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
+			Expect(err).NotTo(HaveOccurred())
+			devNamespace = fw.UserNamespace
+			managedNamespace = utils.GetGeneratedNamespace(managedNamespace)
```

Comment:
Below sentence need to be added  in this block?
`kubeAdminClient = fw.AsKubeAdmin`